src/components/KlayFaucet.js

```diff
@@ -83,6 +77,17 @@ class KlayFaucet extends Component<Props> {
     });
   }
 
+  changeWallet = (e) => {
+    if(e.target.value) {
+      this.wallet.address = e.target.value
```

Comment:
minor comment.
It would be nice if wallet address verification is added.

Reply:
good suggestion. @jeonghonah Could you please take another look?